FWCore/Services/plugins/BuildFile.xml

```diff
@@ -1,6 +1,8 @@
 <use   name="FWCore/Services"/>
 <use   name="Utilities/StorageFactory"/>
+<use   name="boost"/>
 <use   name="gcc-atomic"/>
 <library   file="*.cc" name="FWCoreServicesPlugins">
   <flags   EDM_PLUGIN="1"/>
+  <flags   CXXFLAGS="-Wno-maybe-uninitialized"/>
```

Comment:
if there is a way f disabling the check only in the boost headers, that works for me as well...